login-module/test/core_test.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env mocha */
+import chai from 'chai'
+import dirtyChai from 'dirty-chai'
+import GraphAcademy from '../src/index.js'
+
+const expect = chai.expect
+chai.use(dirtyChai)
+
+describe('GraphAcademy', () => {
```

Comment:
Please note that it's possible to share the same tests suite (or at least some parts) between Node and the browser.